src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu.graphql

```diff
@@ -0,0 +1,9 @@
+query ManageOrganizationsAccess {
+  user {
+    administrativeOrganizations(first: 2) {
```

Comment:
We only check that there is at least one org, so we only need to one, right?

```suggestion
    administrativeOrganizations(first: 1) {
```

Reply:
That is correct, I can change it to 1